System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryTests.cs

```diff
@@ -788,6 +788,20 @@ public void MockDirectory_GetFiles_Returns_Files()
             Assert.AreEqual(testPath, entries.First());
         }
 
+        [Test]
+        public void MockDirectory_GetFiles_Returns_Files_WithRelativePath()
+        {
+            // arrange
+            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>());
+
+            string directory = XFS.Path(@"C:\foo");
+            fileSystem.Directory.SetCurrentDirectory(directory);
+            fileSystem.AddFile($@"C:\test.txt", new MockFileData("Some ASCII text."));
+
+            Assert.AreEqual(fileSystem.Directory.GetFiles(@"C:\").Length, 1); // Assert with absolute path
```

Comment:
I'd remove this assertion, because absolute paths are already checked in the test above.